aeneas/src/mach/MachBackend.v3

```diff
@@ -19,6 +19,33 @@ class MachMoves {
 	var valMoves: List<(Val, int)>; // value -> usepos moves, before
 	var after: MoveResolver;	// phys -> phys moves, after
 }
+// Machine-independent representation of virtual register classes
+enum RegClass(kind: byte) {
```

Comment:
Actually, now that I think about it, we can just use the implicitly-declared "tag" for the register class.
Let's call this first one "REF" as well.

Reply:
You mean the enum tag? Are you saying we should replace the RegClass field in VReg with an int that represents the tag?

Reply:
Done.